addons/account/views/account_view.xml

```diff
@@ -864,6 +864,11 @@
             <field name="arch" type="xml">
                 <form string="Operation Templates">
                     <sheet>
+                        <div class="oe_button_box" name="button_box">
+                            <button type="object" name="action_reconcile_stat"
+                              string="Stat" class="oe_stat_button" icon="fa-info-circle">
```

Comment:
`Open Journal Entries` ?

check with avw for the label + the icon + the number included or not in the label